OpenRA.Mods.Common/Traits/Cargo.cs

```diff
@@ -37,7 +37,7 @@ public class Cargo : IPips, IIssueOrder, IResolveOrder, IOrderVoice, INotifyCrea
 	{
 		public readonly CargoInfo Info;
 		readonly Actor self;
-		readonly List<Actor> cargo = new List<Actor>();
+		readonly Queue<Actor> cargo = new Queue<Actor>();
```

Comment:
I like this in principle. How about a `Stack` instead, though? You'd get that nice LIFO behaviour which would fit best for `Cargo`, I think.